src/main/java/de/zalando/aruha/nakadi/controller/TopicsController.java

```diff
@@ -115,7 +112,7 @@ public TopicPartition getPartition(@PathVariable("topicId") final String topicId
 	}
 
 	@Timed
-	@RequestMapping(value = "/{topicId}/partitions/{partitionId}/events", method = RequestMethod.GET)
+	@RequestMapping(value = "/{topicId}/partitions/{partitionId}/events/one", method = RequestMethod.GET)
```

Comment:
I assume this is a test-endpoint as it's not really part of API, so I suggest to have just a hacky name for it because it is occupying the URL that we should have for streaming endpoint.